pyramid_oereb/lib/records/plr.py

```diff
@@ -62,31 +61,24 @@ def __init__(self, theme, information, law_status, published_from, responsible_o
         """
         super(PlrRecord, self).__init__(theme)
 
-        if not isinstance(information, dict):
-            warnings.warn('Type of "information" should be "dict"')
+        if not isinstance(legende_text, dict):
+            warnings.warn('Type of "legende_text" should be "dict"')
 
         if sub_theme is not None and not isinstance(sub_theme, dict):
             warnings.warn('Type of "sub_theme" should be "dict"')
 
         assert isinstance(geometries, list)
         assert len(geometries) > 0
 
-        self.information = information
+        self.legende_text = legende_text
         self.law_status = law_status
         self.published_from = published_from
+        self.published_until = published_until
```

Comment:
If you add this you maybe should link and solve also issue #1266 in your pull request

Reply:
Ok.